docs/contributing/developing.rst

```diff
@@ -179,6 +185,28 @@ IE             Desktop        10 and below
 Safari         Windows        All
 =============  =============  =============
 
+Accessibility targets
+---------------------
+
+We want to make Wagtail accessible for users of a wide variety of assistive technologies. The specific standard we aim for is `WCAG2.1 <https://www.w3.org/TR/WCAG21/>`_, AA level. Wagtail’s administration user interface isn’t accessible at the moment (see `issue #4199 <https://github.com/wagtail/wagtail/issues/4199>`_), but here are specific assistive technologies we aim to test for, and ultimately support:
+
+=============  ====================
+Type           Assistive technology
+=============  ====================
+Screen reader  `NVDA <https://www.nvaccess.org/download/>`_ on Windows with Firefox ESR
+Screen reader  `VoiceOver <https://support.apple.com/en-gb/guide/voiceover-guide/welcome/web>`_ on macOS with Safari
+Magnification  `Windows Magnifier <https://support.microsoft.com/en-gb/help/11542/windows-use-magnifier>`_
+Magnification  macOS Zoom
+Voice control  Windows Speech Recognition
+Voice control  macOS Dictation
```

Comment:
Should we mention a mobile screenreader here? My experience is that VoiceOver works near identically on iOS and MacOS so maybe we could include an Android offering? Or is that biting off too much for now...

Reply:
_(for reference, these choices are based on the UK’s GDS [2016 assistive technology survey](https://accessibility.blog.gov.uk/2016/11/01/results-of-the-2016-gov-uk-assistive-technology-survey/), with further guidance on what to test with if you want to do [testing at no cost](https://accessibility.blog.gov.uk/2018/09/27/assistive-technology-tools-you-can-use-at-no-cost/))_

I'm in two minds about it. It feels really relevant for mobile screenreader(s) / accessibility tools to be on there, and I regularly use my phone's VoiceOver when doing testing. At the same time, Wagtail’s mobile experience is quite poor for sighted users, so aiming for screen reader support there feels like a step too far. I don't know how to decide :/